app/mesh/1.2.x/features/opa.md

```diff
@@ -0,0 +1,355 @@
+    ```
+
+1.  Pass the Secret to `OPAPolicy`:
+
+    ```yaml
+    type: OPAPolicy
+    mesh: default
+    name: opa-1
+    selectors:
+    - match:
+        kuma.io/service: '*'
+    conf:
+      policies:
+        - secret: opa-policy
+    ```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Configuration
+
+Kong Mesh defines a default configuration for OPA, but you can adjust the configuration to meet your environment's requirements.
+
+The following environment variables are available:
+
+| Variable                   | Type      | What it configures     | Default value {:width=25%:}   |
+| -------------------------- | --------- | --------------------------------------| ------------------- |
+| KMESH_OPA_ADDR             | string    | Address OPA API server listens on     | `localhost:8181`    |
+| KMESH_OPA_CONFIG_PATH      | string    | Path to file of initial config        | N/A                 |
+| KMESH_OPA_DIAGNOSTIC_ADDR  | string    | Address of OPA diagnostics server     | `0.0.0.0:8282`      |
+| KMESH_OPA_ENABLED          | bool      | Whether `kuma-dp` starts embedded OPA | true                |
+| KMESH_OPA_EXT_AUTHZ_ADDR   | string    | Address of Envoy External AuthZ service | `localhost:9191`  |
+| KMESH_OPA_CONFIG_OVERRIDES | strings   | Overrides for OPA configuration, in addition to config file(*) | [plugins.envoy_ext_authz_grpc. query=data.envoy.authz.allow] |
+
+{% navtabs %}
+{% navtab Kubernetes %}
+
+You can customize the agent in either of the following ways:
+
+- Override variables in the data plane proxy config: 
+{% navtabs %}
+{% navtab kumactl %}
+
+When you deploy the Mesh control plane, edit the `kong-mesh-control-plane-config` ConfigMap:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kong-mesh-control-plane-config
+  namespace: kong-mesh-system
+data:
+  config.yaml: |
+    runtime:
+      kubernetes:
+        injector:
+          sidecarContainer:
+            envVars:
+              KMESH_OPA_ENABLED: "false"
+              KMESH_OPA_ADDR: ":8888"
+              KMESH_OPA_CONFIG_OVERRIDES: "config1:x,config2:y"
+```
+
+{% endnavtab %}
+{% navtab Helm %}
+
+Override the Helm value in `values.yaml`
+
+```yaml
+kuma:
+  controlPlane:
+    config: |
+      runtime:
+        kubernetes:
+          injector:
+            sidecarContainer:
+              envVars:
+                KMESH_OPA_ENABLED: "false"
+                KMESH_OPA_ADDR: ":8888"
+                KMESH_OPA_CONFIG_OVERRIDES: "config1:x,config2:y"
+```
+
+{% endnavtab %}
+{% endnavtabs %}
```

Comment:
I don't think you need the endnavtabs here yet right? And you only need one endnavtab - I think.